stage_descriptions/sorted-sets-10-fu7.md

```diff
@@ -0,0 +1,63 @@
+In this stage, you'll add support for removing multiple zset members using the `ZREM` command.
```

Comment:
(Added note in stage description but repeating here - let's remove this altogether)